lib/AST/ASTContext.cpp

```diff
@@ -1739,7 +1740,9 @@ ASTContext::getModule(ArrayRef<std::pair<Identifier, SourceLoc>> ModulePath) {
     if (ModuleDecl *M = importer->loadModule(moduleID.second, ModulePath)) {
       if (ModulePath.size() == 1 &&
           (ModulePath[0].first == StdlibModuleName ||
-           ModulePath[0].first == Id_Foundation))
+           ModulePath[0].first == Id_Foundation ||
+           // SWIFT_ENABLE_TENSORFLOW
+           ModulePath[0].first == getIdentifier("TensorFlow")))
```

Comment:
Add `ID_TensorFlow`?

Reply:
Done. There are many usages of `getIdentifier("TensorFlow")`, I'll replace them in a follow-up.